### PR TITLE
Update and add missing package comments

### PR DESCRIPTION
**Pull request by Jacalz**
### Description:

This updates some of the package comments and adds some new ones. After this change, we have package comments for every public library package. 

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [ ] Tests all pass.

### CODE REVIEW

**Comment by Jacalz** [comment]
Feel free to come with suggestions if you have ideas for better package comments. Every improvement counts.

**Review by andydotxyz — APPROVED** [review]

File: test/testapp.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Package test provides utility drivers for running UI tests without rendering
+// Package test provides utility drivers for running UI tests without and without rendering.
```
Comment (stuartmscott) [inline test/testapp.go:1]: > // Package test provides utility drivers for running UI tests without and without rendering.

Duplicate "without"

```suggestion
// Package test provides utility drivers for running UI tests without rendering to a screen.
```

**Review by stuartmscott — CHANGES_REQUESTED** [review]

File: test/testapp.go (hunk 1 above)
Comment (Jacalz) [reply test/testapp.go]: A bit late of a comment, but thanks. My intent was to say "with or without" rendering, but your suggestion was even better :)

Comment (stuartmscott) [reply test/testapp.go]: "A bit late of a comment"?

Comment (Jacalz) [reply test/testapp.go]: I meant that I committed your change early this morning but sent the comment many hours later.

**Review by stuartmscott — APPROVED** [review]

**Pull request merged**